### DISCUSSION TITLE
OKD and FCOS 39

### DISCUSSION THREAD

**Discussion by forzamehlano** [okd-project/okd#1834] [General]
Are there any plans/timescales on OKD 4 utilising FCOS 39? I'd assumed (looking back over OKD 4.12/4.13 at least) that each OKD minor version incremented the FCOS version too so was expecting FCOS 39 with OKD 4.14. 

The releases page suggests 4.14 still has FCOS 38

I see the audit package is now part of 39 which is something we've been waiting for and wondered how long that will take to filter down to OKD.

**Comment by vrutkovs** [comment]
OKD 4.15 will use F39. So far I'm cautious about updating 4.14 to F38, especially at the end of the year.

Lets bring that up at OKD WG meeting on Jan 9?
